packages/@aws-cdk/aws-lambda-nodejs/test/builder.test.ts

```diff
@@ -67,15 +67,3 @@ test('throws if status is not 0', () => {
   });
   expect(() => builder.build()).toThrow('status-error');
 });
-
-test('throws when parcel-bundler is not 1.x', () => {
```

Comment:
Add a test for when `docker` is not installed.

Reply:
Done